grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -491,7 +499,32 @@ def create_register_file(self, filename):
                         br='S2_{}'.format(band_ref)
                     ))
                 fd.write(os.linesep)
+
+    def set_footprint(self,fp):
+        reg_name = "tmpregion_%s"%(str(os.getpid()))
+        gs.run_command('g.region',save=reg_name)
+        fp_dict={}
+        idlist = [x for x in gs.parse_command('v.db.select',map=fp)]
+        cat_idx = idlist[0].split('|').index('cat')
+        ident_idx = idlist[0].split('|').index('identifier')
+
+        for item in idlist[1:]:
+            tmp_key = item.split('|')[ident_idx]
+            key = "%s_%s"%(tmp_key.split('_')[5],tmp_key.split('_')[2])
+            fp_dict[key] = item.split('|')[cat_idx]
+        fp_rast_name = "fp_rast_%s"%(str(os.getpid()))
+        for map in imported_map_list:
+            gs.run_command('g.region',raster=map)
```

Comment:
Here you could do:
```
reg = gs.parse_command("v.db.select", flags="r", map=fp, where="cats in (fp_dict['%s_%s'%(map.split('_')[0],map.split('_')[1])])")
gs.use_temp_region()
gs.run_command('g.region', align=map, n=reg["n"], s=reg["s"], e=reg["e"], w=reg["w"])
```
That zooms the computational region to the footprint, but Pixel alignment follows the input map

Reply:
now included (see above)